__mocks__/react-relay.js

```diff
@@ -1,4 +1,4 @@
-let React = require.requireActual("react")
+let React = jest.requireActual("react")
```

Comment:
`require.requireActual` is no longer available anymore

Reply:
Interesting that they removed this! I wonder why.